listenbrainz/webserver/views/test/test_user.py

```diff
@@ -41,3 +62,17 @@ def test_scraper_username(self):
         )
         self.assert200(response)
         self.assertIn('var user_name = "weird%5Cuser%20name";', response.data.decode('utf-8'))
+
+    def _create_test_data(self, user_name):
+        test_data = create_test_data_for_influxlistenstore(user_name)
+        self.logstore.insert(test_data)
+
+    def test_username_case(self):
+        """Tests that the username in URL is case insenstive"""
+        self._create_test_data(self.user['musicbrainz_id'])
```

Comment:
Making the user_name explicit here might be a good idea, considering we use the user name (and a modified version) later.

`self._create_test_data('iliekcomputers')`